main.py

```diff
@@ -2,6 +2,41 @@
 
 from django.db.models import QuerySet
 
+from db.models import Genre, Actor
+
 
 def main() -> QuerySet:
-    pass
+    Genre.objects.bulk_create(
+        [Genre(name="Western"),
+         Genre(name="Action"),
+         Genre(name="Dramma")]
```

Comment:
There is a typo in the genre name 'Dramma'. It should be 'Drama'. This is corrected later in the code, but it's better to avoid introducing incorrect data initially.